### PR TITLE
@craigspaeth => Don't crash on nil show names

### PR DESCRIPTION
**Pull request by mzikherman**

### CODE REVIEW

File: schema/show.js [hunk 1]
```diff
@@ -70,7 +70,8 @@ const ShowType = new GraphQLObjectType({
     name: {
       type: GraphQLString,
       description: 'The exhibition title',
-      resolve: ({ name }) => name.trim(),
+      resolve: ({ name }) =>
+        isExisty(name) ? name.trim() : '',
```
Comment (alloy) [inline schema/show.js]: Why return an empty string instead of `null`?

Comment (mzikherman) [inline schema/show.js]: Good q - I wasn't sure if clients are displaying it and being `null` might be worse (than a blank string)? I also didn't think it was necessary, i can undo.

Comment (alloy) [inline schema/show.js]: Seeing as the schema indicates that this field may be `null` (i.e. it’s not designated not-null), clients should already be aware of that and use the `null` sentinel as a way to determine what to do. I think `null` is better than an empty string for those cases.

File: schema/show.js [hunk 2]
```diff
@@ -71,7 +71,7 @@ const ShowType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'The exhibition title',
       resolve: ({ name }) =>
-        isExisty(name) ? name.trim() : '',
+        isExisty(name) ? name.trim() : name,
```
Comment (alloy) [inline schema/show.js]: FYI, I think the JS idiom is to do `name && name.trim()`, but that’s not a blocker imo

**Comment by alloy** [comment]
Seeing as there is little to this PR and I already reviewed it, I’m taking this one.

**Comment by craigspaeth** [comment]
Thanks for taking this Eloy!

**Pull request merged**